### PR TITLE
Post Carousel: Improve Multiple Carousel and Mobile Support

### PR DESCRIPTION
**Pull request by AlexGStapleton**
This PR improves support for multiple post carousels on the same page but setting variables as needed. This ensures they're correct and relevant when used.

### CODE REVIEW

**Comment by Misplon** [comment]
@AlexGStapleton Please, check, can you scroll past 10 posts on mobile when posts per page isn't set?

**Comment by Misplon** [comment]
I've confirmed locally, on mobile, it isn't possible to scroll past 10 posts. My test layout:

[24785.json.zip](https://github.com/siteorigin/so-widgets-bundle/files/4955764/24785.json.zip)

This layout uses the latest [unit test theme data](https://codex.wordpress.org/Theme_Unit_Test). The categories I've used are `category: block` and `cateogry: classic`, the only two categories I have with more than 10 posts.

**Comment by Misplon** [comment]
@AlexGStapleton There is an edge case where the 10 hard (can't scroll past) limit is being applied. Resize your browser as small as it'll go and then grab posts with your mouse cursor, I can't get past 10 in that situation.

File: widgets/post-carousel/js/carousel.js [hunk 1]
```diff
@@ -23,7 +17,7 @@ jQuery( function ( $ ) {
 				arrows: false,
 				infinite: false,
 				rows: 0,
-				rtl: direction == 'rtl',
+				rtl: $$.attr( 'dir' ) == 'rtl',
```
Comment (gregpriday) [inline widgets/post-carousel/js/carousel.js:20]: I missed this before, but shouldn't this be a data attribute? So `data-dir`?

Comment (Misplon) [reply widgets/post-carousel/js/carousel.js:20]: Resolved via https://github.com/siteorigin/so-widgets-bundle/commit/f0ae7eed63c7f5a17e2e19070f98a9cdaf6cfdf2.

**Pull request merged**